### ISSUE TITLE
Can I use Apple Pay for auto-renewable subscription via this library?

### ISSUE THREAD

**Issue by goleador0322** [chirag04/react-native-in-app-utils#172]
Can I use Apple Pay for auto-renewable subscription via this library?
And if I can, how can I use it.

**Comment by philwilks** [comment]
Nope, this is for in-app purchases from the App Store. This is for are very different to Apple Pay which is generally a credit card substitute for purchasing things like physical goods.

**Comment by goleador0322** [comment]
I see ~~~~! thanks :)
